exporter/stats/stackdriver/stackdriver.go

```diff
@@ -136,6 +142,16 @@ func (e *Exporter) Export(vd *stats.ViewData) {
 	}
 }
 
+// getTaskValue returns a task label value in the format of
+// "go-<pid>@<hostname>".
+func getTaskValue() string {
```

Comment:
This assumes you can only run one Stackdriver exporter per process.

Add a note about that?

Reply:
That's right, actually in Java we enforce exporter to be singleton: https://github.com/census-instrumentation/opencensus-java/blob/master/exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverStatsExporter.java#L261

I added a note about this for NewExporter().